### ISSUE TITLE
Don't open the gallery, if you click on thumbnails in a post with uploaded pictures

### ISSUE THREAD

**Issue by kas70** [diaspora/diaspora#6649]
My proposal:
- open the gallery if you click on the maximized image only (blue circle)
- change the maximized image, if you click on the small thumbnails below (red circle => replace the maximized pic with the one you clicked on)
![pics_stream](https://cloud.githubusercontent.com/assets/922628/12530076/42c83a80-c1d2-11e5-85ac-e058bde6913a.png)

I think this would improve the stream reading experience, because it's less disrupting than always showing the gallery, which overlays the whole page.

**Comment by Flaburgan** [comment]
I agree there is something to do here to improve user experience. Not sure your proposal is the way to go though, it doesn't look intuitive to me to open *another* picture when you're clicking on one. Maybe we could add a left and right arrows next to the big picture to allow the user to change it without open the gallery.

**Comment by jaywink** [comment]
I really like this idea. Many web shops work like this, for example :+1:

**Comment by kas70** [comment]
What do you mean with '*open another picture*'? I'd just show every pic as a thumbnail at the bottom and show the selected above (default is the first one in the row).
Take a look at this example gallery, which has my proposed behaviour (without the option to open the full-page gallery):
https://johnpolacek.github.io/ResponsiveThumbnailGallery/

**Comment by kas70** [comment]
Also: although i only was searching for a bad example, the gallery layout looks quite nice (especially without the a space between the thumbnails and the big image).

**Comment by Flaburgan** [comment]
>What do you mean with 'open another picture'?

Sorry I misunderstood what you said, never mind.
